reddit.js

```diff
@@ -1,5 +1,38 @@
 var reddit = require("./data.json")
+// console.log(reddit["data"]);
 
 // Write your solutions below
 
-console.log(reddit["data"]);
\ No newline at end of file
+// forEach - cosole.log all titles
+reddit.data.children.forEach(function (titles) {
+	console.log(titles.data.title);
+});
+
+// map - return an array of permalinks, append http://reddit.com to each so they contain full http:// path to reddit
+var permalinks = reddit.data.children.map(function (link) {
+	return link.data.permalink;
+});
+
+console.log(permalinks);
+
+permalinks.forEach(function (url) {
+	console.log("http://reddit.com" + url);
+});
```

Comment:
Nice use of forEach here. Your parameter name is accurately descriptive as well.